packages/db/prisma/schema.prisma

```diff
@@ -10,8 +10,8 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Config {
-  id          Int         @id @default(autoincrement())
+model IndexerMetadata {
```

Comment:
Changing the schema requires committing the prisma migration to the repository.

Can you try `npx prisma migrate dev --schema ./packages/db/prisma/schema.prisma`?

Reply:
```
api_1      | → 18   const indexerMetadata = await ctx.prisma.indexerMetadata.findUnique(
api_1      | The table `public.IndexerMetadata` does not exist in the current database.
api_1      | prisma:error 
api_1      | Invalid `ctx.prisma.indexerMetadata.findUnique()` invocation in
api_1      | /app/packages/api/src/router/indexer.ts:18:64
api_1      | 
api_1      |   15 .input(z.void())
api_1      |   16 .output(z.object({ slot: z.number() }))
api_1      |   17 .query(async ({ ctx }) => {
api_1      | → 18   const indexerMetadata = await ctx.prisma.indexerMetadata.findUnique(
api_1      | The table `public.IndexerMetadata` does not exist in the current database.
api_1      | prisma:error 
api_1      | Invalid `ctx.prisma.indexerMetadata.findUnique()` invocation in
api_1      | /app/packages/api/src/router/indexer.ts:18:64
api_1      | 
api_1      |   15 .input(z.void())
api_1      |   16 .output(z.object({ slot: z.number() }))
api_1      |   17 .query(async ({ ctx }) => {
api_1      | → 18   const indexerMetadata = await ctx.prisma.indexerMetadata.findUnique(
api_1      | The table `public.IndexerMetadata` does not exist in the current database.
api_1      | prisma:error 
api_1      | Invalid `ctx.prisma.indexerMetadata.findUnique()` invocation in
api_1      | /app/packages/api/src/router/indexer.ts:18:64
api_1      | 
api_1      |   15 .input(z.void())
api_1      |   16 .output(z.object({ slot: z.number() }))
api_1      |   17 .query(async ({ ctx }) => {
api_1      | → 18   const indexerMetadata = await ctx.prisma.indexerMetadata.findUnique(
api_1      | The table `public.IndexerMetadata` does not exist in the current database.
```